api/lib/resolvers.js

```diff
@@ -69,8 +78,26 @@ async function validators(
   return validators
 }
 
-async function validator(_, { networkId, operatorAddress }, { dataSources }) {
+async function validator(
+  _,
+  { networkId, operatorAddress, getFeed },
+  { dataSources }
+) {
   await localStore(dataSources, networkId).dataReady
+  if (getFeed) {
```

Comment:
this doesn't make sense. for 1 validator you are loading all validators feeds. what did we gain by that? instead just load the feed for this 1 validator.

Why do we need the property `getFeed`? Instead use a `resolver` like `expectedReturns` above. So when ever the `feed` property is requested, it is loaded but only if it is requested. But the distinction makes no sense as we will always load the feed when querying for just 1 validator as this query is only happening on the validator page.

Reply:
OK, I was looking to make the query more efficient, but guess I can use the `_in` operator with one address as well

Reply:
> But the distinction makes no sense

So I just remove any check then?

Reply:
>use a resolver like expectedReturns above. So when ever the feed property is requested, it is loaded but only if it is requested.

also change the validator feed db query to only accept 1 validator address as this is the only case we will have.